tests/indico_wrapper/test_indico_wrapper.py

```diff
@@ -1,20 +1,25 @@
-from indico import IndicoClient, types
-from indico.queries import CreateStorageURLs
+import pytest
```

Comment:
could probably write a test for retry here by making some dummy function that raises an exception and increments some counter and can then validate that the counter is incremented to the correct #